e2e/operator_scheduler/input/basic.nomad

```diff
@@ -0,0 +1,21 @@
+job "operator_scheduler" {
+  datacenters = ["dc1"]
+  type        = "service"
+
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
+
+  group "operator_scheduler" {
+
+    task "test" {
+      driver = "raw_exec"
+
+      config {
+        command = "bash"
+        args    = ["-c", "sleep 15000"]
```

Comment:
It might be worth dropping this down to 1s and making it a batch job so that we can guarantee a quick exit once we've turned the broker back on; that'll reduce the risk of test flakiness on the `WaitForJobStop` (which definitely _should_ be enough time, but...)